### PR TITLE
Fixes violations of Style/ConstantName and reenables the check.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: bin/cron/activity-monitor [hunk 1]
```diff
@@ -194,18 +194,18 @@ end
 def main()
   now = DateTime.now.new_offset(0)
 
-  hours = HoursToTest
+  hours = HOURS_TO_TEST
```
Comment (ashercodeorg) [inline bin/cron/activity-monitor:197]: Note the variable hours seems superfluous, I'll remove it in a followup PR.

**Comment by joshlory** [comment]
What is the benefit of this cop?

**Comment by ashercodeorg** [comment]
So that constants are easily identifiable as constants.

**Comment by ashercodeorg** [comment]
Again, I'll drop and regenerate as that will be easier than the merge conflict. Do you see this cop as being worth enforcing (I do)? Your earlier comment suggests no.

**Pull request closed without merging**